src/foundry/foundry.js/clientDocuments/combat.d.ts

```diff
@@ -16,7 +18,202 @@ declare global {
      * The configuration setting used to record Combat preferences
      */
     static CONFIG_SETTING: 'combatTrackerConfig';
```

Comment:
this should go at the correct place according to the order of declaration, i.e. after `_soundPlaying`